vividus-tests/src/main/resources/story/integration/ScopeVariable.story

```diff
@@ -0,0 +1,7 @@
+Description: Integration tests for scope variables
+
+Meta:
+    @group scope-variables
+
+Scenario: Verify batch scope has higher priority than global scope
+Then `${scopeName}` is equal to `batch-1-value`
```

Comment:
What about moving to https://github.com/vividus-framework/vividus/blob/master/vividus-tests/src/main/resources/story/integration/VariableTests.story ?